api/CompositionEvent.json

```diff
@@ -128,10 +128,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "6.1"
```

Comment:
The `data` attribute was there from the start, so something seems to be wrong here. I would suspect that the interface object wasn't exposed initially, but then when was the event first supported?

If that could be answered, data for the 3 composition* events could also be filled out.